features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/ImageTypeResolver.kt

```diff
@@ -9,17 +9,18 @@ package com.datadog.android.sessionreplay.internal.recorder.base64
 import android.graphics.drawable.Drawable
 import android.graphics.drawable.GradientDrawable
 import androidx.annotation.VisibleForTesting
+import com.datadog.android.sessionreplay.internal.recorder.densityNormalized
 
 internal class ImageTypeResolver {
-    fun isDrawablePII(drawable: Drawable, widthInDp: Int, heightInDp: Int): Boolean =
+    fun isDrawablePII(drawable: Drawable, density: Float): Boolean =
```

Comment:
> [!NOTE]
> For readability, it might be worth making this a block function `fun isDrawable(…): Boolean { … }` and naming each boolean contributing to the condition so that we can better understand the logic.
> e.g.: `return notGradient && (widthAboveThreshold || heightAboveThreshold)`